docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -736,6 +736,25 @@ The output of the query has the following columns:
     - ``boolean``
     - Whether or not the operation appended data
 
+Functions
+---------------
+
+The connector exposes following functions:
+
+table_changes
+^^^^^^^^^^^^^
+
+Allows reading CDF entries:
+
+.. code-block:: sql
+
+    SELECT * FROM TABLE(system.table_changes('test_schema.test_table', startVersion, endVersion)
+
+``startVersion`` - type ``BIGINT``, exclusive, changes will be read from ``startVersion + 1``
```

Comment:
I like sinceVersion, @alexjo2144 ?

Reply:
Does it matter? Since it's a function for reading *changes*, it necessarily has to be the delta between versions (i.e., what changes are needed to transition from start -> end), so it's expected for the start version to be exclusive (it's the base snapshot), and end version to be inclusive (it's the state we're trying to get to)

BTW, use snake case for the names: `start_version`, `end_version`.

Also, I would go with `from_version` and `to_version`, as in "get me the changes from this version of the table to this version"